snitun/server/listener_peer.py

```diff
@@ -17,20 +19,20 @@
 class PeerListener:
     """Peer Listener class."""
 
-    def __init__(self, peer_manager: PeerManager, host=None, port=None):
+    def __init__(self, peer_manager: PeerManager, host:str|None=None, port:int|None=None) -> None:
```

Comment:
For the simple reason that I did not add spaces while writing them, and my env did not have a linter to automatically adjust it because of the current work with switching it out and the conflicts it causes :)

Reply:
Adjusted in https://github.com/NabuCasa/snitun/pull/242/commits/4865abd0052183cb54829339b8238dafebe8fb16